episodes/python-template.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Generating a Repository using a Template"
+teaching: 10
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How can you quickly set up a new Python repository that follows best practices?
```

Comment:
If we connect it also to the SMP we can add the question:
How to create a new Python repository that covers your needs and follows best practices